src/main/webapp/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```

Comment:
In the past, Vidhya has told me not to push test html files to git. You should probably do the same. (Use it locally and save it somewhere else and delete when you submit)